lib/cacheconfiguration.h

```diff
@@ -72,12 +75,22 @@ class CONTACTCACHE_EXPORT CacheConfiguration : public QObject
     MGConfItem m_displayLabelOrderConf;
     MGConfItem m_sortPropertyConf;
     MGConfItem m_groupPropertyConf;
+#endif
+
+#ifdef HAS_GSETTINGS
+    QGSettings *m_propertyConf;
+#endif // HAS_GSETTINGS
 
+#ifdef HAS_MLITE
 private slots:
     void onDisplayLabelOrderChanged();
     void onSortPropertyChanged();
     void onGroupPropertyChanged();
 #endif
+#ifdef HAS_GSETTINGS
+private slots:
```

Comment:
Minor detail but could take the private slots to common part? Alternatively if the new connect() syntax with method pointers was used, there wouldn't be need for declaring such a thing.